README.md

```diff
@@ -36,6 +36,21 @@ Takes as input a transaction (signed or unsigned) and broadcast it to the networ
 ```
 echo $TX_JSON | lto broadcast
 ```
+### Balance
+
+Display the balances of the default address or if specified of a specific address
+```
+lto balance
+```
+or
+```
+lto balance 3JuijVBB7NCwCz2Ae5HhCDsqCXzeBLRTyeL --network T --available
+```
+Options: 
+- regular
+- generating
+- available
+- effective
```

Comment:
We're also not putting the options for other commands in the README.

Reply:
the example has the --available option, should i also remove that option from the example and leave 
lto balance 3JuijVBB7NCwCz2Ae5HhCDsqCXzeBLRTyeL --network T 
instead of
lto balance 3JuijVBB7NCwCz2Ae5HhCDsqCXzeBLRTyeL --network T --available

Reply:
yeah maybe. Then also remove the `--network`